tests/test_bgp_neighbor_af.rb

```diff
@@ -150,15 +233,17 @@ def test_props_bool
   def props_bool(af, dbg)
     # These properties have simple boolean states. As such we can use a common
     # set of tests to validate each property.
+    # Common to both XR and nexus
     props = [
       :as_override,
-      :disable_peer_as_check,
       :next_hop_self,
-      :next_hop_third_party,
-      :suppress_inactive,
     ]
 
-    props = [:disable_peer_as_check] if dbg.include?('l2vpn/evpn')
+    if platform == :nexus
+      props << :next_hop_third_party
+      props << :suppress_inactive
+      props = [:disable_peer_as_check] if dbg.include?('l2vpn/evpn')
```

Comment:
Yes exactly, I tested it pretty thoroughly. For XR the only properties supported are:
 :as_override,
 :next_hop_self,

Under all af's.

For Nexus they are all supported, except when a l2vpn in which case the only disable_peer_as_check is supported.